lib/puppet/provider/reboot/linux.rb

```diff
@@ -1,52 +0,0 @@
-Puppet::Type.type(:reboot).provide :linux do
```

Comment:
Have we released a version of this module with `linux` as the provider? If so, then this is a breaking change since it will break anyone doing:

```
reboot { '...':
  provider => linux
}
```

In which case, you'll want to add the old one back (so you have both linux and posix) and add a deprecation warning to the linux one.